src/OpenLoco/src/Windows/LandscapeGeneration.cpp

```diff
@@ -485,16 +485,18 @@ namespace OpenLoco::Ui::Windows::LandscapeGeneration
                 drawingCtx.drawImage(&rt, 2, yPos + 1, imageId);
 
                 // Draw land description.
-                _commonFormatArgs[0] = landObject->name;
-                drawingCtx.drawStringLeftClipped(rt, 24, yPos + 5, 121, Colour::black, StringIds::wcolour2_stringid, &*_commonFormatArgs);
+                FormatArguments args{};
+                args.push(landObject->name);
+                drawingCtx.drawStringLeftClipped(rt, 24, yPos + 5, 121, Colour::black, StringIds::wcolour2_stringid, &args);
 
                 // Draw rectangle.
                 drawingCtx.fillRectInset(rt, 150, yPos + 5, 340, yPos + 16, window.getColour(WindowColour::secondary), Drawing::RectInsetFlags::borderInset | Drawing::RectInsetFlags::fillDarker);
 
                 // Draw current distribution setting.
                 const StringId distributionId = landDistributionLabelIds[enumValue(S5::getOptions().landDistributionPatterns[i])];
-                _commonFormatArgs[0] = distributionId;
-                drawingCtx.drawStringLeftClipped(rt, 151, yPos + 5, 177, Colour::black, StringIds::black_stringid, &*_commonFormatArgs);
+                args.rewind();
```

Comment:
If you just wrapped the draw call and args in {} above then you would need to rewind. These two args uses are not really related so ideally rewind shouldn't be used